sdk/lib/ucrt/string/strnlen.cpp

```diff
@@ -78,11 +78,13 @@ static __forceinline size_t __cdecl common_strnlen_c(
 
 #ifdef _CRT_SIMD_SUPPORT_AVAILABLE
 
+_UCRT_ENABLE_EXTENDED_ISA
+
     template <strnlen_mode Mode, __crt_simd_isa Isa, typename Element>
     _Check_return_
     _When_(maximum_count > _String_length_(string), _Post_satisfies_(return == _String_length_(string)))
     _When_(maximum_count <= _String_length_(string), _Post_satisfies_(return == maximum_count))
-    static __forceinline size_t __cdecl common_strnlen_simd(
+    static __inline size_t __cdecl common_strnlen_simd(
```

Comment:
Why the change? Does GCC complain even if we have suitable `#define __forceinline` defines available?
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=__forceinline

Reply:
GCC cannot inline functions that use a different cpu target. inline will simply be ignored, but __forceinline is an error.